build.gradle.kts

```diff
@@ -33,45 +33,60 @@ configurations {
-    implementation(group = "org.springframework.boot", name = "spring-boot-starter-oauth2-resource-server")
-    implementation(group = "org.springframework.boot", name = "spring-boot-starter-jdbc")
-    developmentOnly(group = "org.springframework.boot", name = "spring-boot-devtools")
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.springframework.boot:spring-boot-starter-security")
+    implementation("org.springframework.boot:spring-boot-configuration-processor")
+    implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
+    implementation("org.springframework.boot:spring-boot-starter-jdbc")
+    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+    developmentOnly("org.springframework.boot:spring-boot-devtools")
 
     // Swagger
-    implementation(group = "org.springdoc", name = "springdoc-openapi-starter-webmvc-ui", version = "2.6.0")
+    implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.6.0")
 
     // Util
-    implementation(group = "org.apache.commons", name = "commons-csv", version = "1.14.0")
-    
+    implementation("org.apache.commons:commons-csv:1.14.0")
+
     // Lombok
-    compileOnly(group = "org.projectlombok", name = "lombok")
-    annotationProcessor(group = "org.projectlombok", name = "lombok")
+    compileOnly("org.projectlombok:lombok")
+    annotationProcessor("org.projectlombok:lombok")
 
     // Test
-    testImplementation(group = "org.springframework.boot", name = "spring-boot-starter-test")
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
+    testImplementation("org.junit.jupiter:junit-jupiter")
+    testImplementation("org.mockito:mockito-core")
+    testImplementation("org.mockito:mockito-junit-jupiter")
+    testImplementation("org.testcontainers:junit-jupiter")
+    testImplementation("org.testcontainers:postgresql")
+    testImplementation("org.springframework.security:spring-security-test")
+    testImplementation("io.rest-assured:spring-mock-mvc")
 
-    compileOnly("com.google.code.findbugs:jsr305:3.0.2")
+    // JWT
+    implementation("io.jsonwebtoken:jjwt-api:0.11.5")
+    implementation("io.jsonwebtoken:jjwt-impl:0.11.5")
+    implementation("io.jsonwebtoken:jjwt-jackson:0.11.5")
 
     // Database
     runtimeOnly("org.postgresql:postgresql")
     implementation("org.liquibase:liquibase-core")
 }
 
-tasks {
-    withType<Test> {
-        useJUnitPlatform()
-        systemProperty("spring.profiles.active", "test")
-    }
-    compileJava {
-        dependsOn(openApiGenerate)
-    }
-}
-
+val jacocoExcludedClasses = listOf(
+    "**/dto/**",
+    "**/entity/**",
+    "**/config/**",
+    "**/exception/**",
+    "**/enums/**",
+    "**/*Request*",
+    "**/*Response*",
+    "**/*Exception*",
+    "**/*Application*",
+    "**/online/rabko/api/**",
+    "**/online/rabko/model/**"
```

Comment:
we could use more generic exclutions like `**/model/**` instead of `**/online/rabko/model/**`